File: Golang/ReverseInteger/ReverseInteger_test.go
package ReverseInteger

import "testing"

func TestReverse(t *testing.T) {

	if reverse(123) == 321 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if reverse(-123) == -321 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if reverse(1000000003) == 0 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if reverse(1534236469) == 0 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

}

File: Python/ReverseIntegerTest.py
from unittest import TestCase

from ReverseInteger import ReverseInteger


class TestReverseInteger(TestCase):
    def test_reverse(self):
        ri = ReverseInteger()

        self.assertTrue(ri.reverse(123) == 321)

        self.assertTrue(ri.reverse(-123) == -321)

        self.assertTrue(ri.reverse(1000000003) == 0)

        self.assertTrue(ri.reverse(1534236469) == 0)
